src/wdio/services/TerraCommands/viewport-helpers.js

```diff
@@ -59,6 +59,7 @@ const describeViewports = (title, viewports, fn) => {
   localViewports.forEach(viewport => global.describe(`[${viewport}]`, () => {
     global.before(() => setViewport(viewport));
     global.describe(title, fn);
+    global.after(() => setViewport(formFactor));
```

Comment:
This should restore the to the previous viewport size, not necessarily the formFactor defined in the test configuration

See: https://github.com/cerner/terra-toolkit/blob/master/src/wdio/services/TerraCommands/axe-command.js#L87

Reply:
Resolved this here : https://github.com/cerner/terra-toolkit/pull/363/commits/5cc921af7db23215348e91661e8cdf3e9597744c